pkg/nacler/keyers/secureenclave/secureenclave.go

```diff
@@ -0,0 +1,194 @@
+//go:build darwin
+// +build darwin
+
+package secureenclave
+
+// this was heavily inspiried by https://github.com/facebookincubator/sks
+// thank you!
+
+/*
```

Comment:
Curious -- why do we have some obj-c in the comment here and the rest in `secureenclave.m`? (Is there a convention for how to split it up?) I see it's how the `sks` repo linked above does it too.

Reply:
Seph also called this out, moved to .h file